### DISCUSSION TITLE
Hyperspi ESP32-S2

### DISCUSSION THREAD

**Discussion by jt7005** [awawa-dev/HyperSPI#23] [Q&A] [answered]
Quick question - is there any advantage to using ESP32-S2 over an esp8266 for a setup of 200 led and a Raspberry Pi 3 b+

**Comment by awawa-dev** [comment, marked as answer]
As you can see in the benchmark on the main page, due to the limited capabilities of the esp8266 (small SPI frame), the performance of the esp8266 is about 20% lower than the esp32 or esp32-s2. Will the upgrade from esp8266 to esp32-s2 bring a visible change for small 200 LEDs setup? I doubt it.
